bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +268,25 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
+        if valid_email_check:
+            result_message = 'Body contains these invalid emails:\n'
+            invalid_emails = False
+            for line in body:
+                for email in re.findall(r'\S*@\S*', line):
+                    email_check = re.match(r"(^[a-zA-Z0-9_.+-]"
+                                           +r"+@[a-zA-Z0-9-]+\."
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -274,8 +274,8 @@
             for line in body:
                 for email in re.findall(r'\S*@\S*', line):
                     email_check = re.match(r"(^[a-zA-Z0-9_.+-]"
-                                           +r"+@[a-zA-Z0-9-]+\."
-                                           +r"[a-zA-Z0-9-.]+$)", email)
+                                           + r"+@[a-zA-Z0-9-]+\."
+                                           + r"[a-zA-Z0-9-.]+$)", email)
                     # if eamil do not match regex there is no need to
                     # look up dns
                     if email_dns_check and email_check:
```